packages/wonder-blocks-modal/components/modal.vite_test.js

```diff
@@ -0,0 +1,101 @@
+/* global render, driver */
+// @flow
+import * as React from "react";
+
+// TODO(kevinb): fix vite so that it can handle importing modules
```

Comment:
Is there an issue number/link to include in this TODO?

Reply:
I'll create one.